### PR TITLE
changelog & readme updates

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,144 +1,149 @@
...
-* **OS**
-  + Ubuntu `20.04`/`22.04`
-  + CentOS `7`
-  + RHEL `8`/`9`
-  + SLES - `15-SP4`
+## Prerequisites
 
-* [ROCm supported hardware](https://docs.amd.com/bundle/Hardware_and_Software_Reference_Guide/page/Hardware_and_Software_Support.html)
+Refer to our
+[Linux GPU and OS support](https://rocm.docs.amd.com/en/develop/about/compatibility/linux-support.html)
```
Comment (saadrahim) [inline README.md:40]: latest branch?

File: README.md [hunk 2]
```diff
@@ -1,144 +1,149 @@
...
+# ROCm Performance Primitives library
 
-AMD ROCm Performance Primitives (**RPP**) library is a comprehensive high-performance computer vision library for AMD processors with `HIP`/`OpenCL`/`CPU` back-ends.
-
-## Latest Release
-
-[![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/GPUOpen-ProfessionalCompute-Libraries/rpp?style=for-the-badge)](https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/releases)
-
-## Top level design
+AMD ROCm Performance Primitives (**RPP**) library is a comprehensive, high-performance computer
```
Comment (saadrahim) [inline README.md:6]:
```suggestion
AMD ROCm Performance Primitives (RPP) library is a comprehensive, high-performance computer
```
Any reason to bold it?

File: README.md [hunk 3]
```diff
@@ -1,144 +1,149 @@
...
+vision library for AMD processors that have `HIP`, `OpenCL`, or raw `CPU` backends.
```
Comment (saadrahim) [inline README.md:7]:
```suggestion
vision library for AMD processors that have `HIP`, `OpenCL`, or `CPU` backends.
```
drop raw?

File: README.md (hunk 1 above)
Comment (LisaDelaney) [reply README.md:40]: I wasn't sure if we should use the current link or the one we'll use for 6.0 (the file name is different).

File: README.md [hunk 4]
```diff
@@ -155,79 +160,87 @@ doxygen .Doxyfile
...
+## Tested configurations
 
 * Linux distribution
-  + Ubuntu - `20.04` / `22.04`
-  + CentOS - `7`
-  + RedHat - `8` / `9`
-  + SLES - `15-SP4`
+  * Ubuntu - `20.04` / `22.04`
+  * CentOS - `7`
+  * RedHat - `8` / `9`
```
[the remark this answers is not in the document]
Comment (LisaDelaney) [reply README.md:243]: This is under "tested configurations", so I thought that was fine to stay. We have a link to supported OSs in the Prerequisites section.

[the remark this answers is not in the document]
Comment (LisaDelaney) [reply README.md:243]: Actually, the changelog also has supported configurations for every release...I'm okay to remove it here...thoughts?

**Pull request closed without merging**